client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import {Box, Typography, styled, keyframes  } from '@mui/material';
+// import { useInView } from 'react-intersection-observer';
+
+const UkraineTextInfo = styled((props) => <Typography variant="body1" {...props} />)(({ theme }) => ({
```

Comment:
please delete variant = 'body1' as it is included by default

Reply:
done